### ISSUE TITLE
CLI flag to run transcripts in parallel

### ISSUE THREAD

**Issue by ChrisPenner** [unisonweb/unison#2653]
Low priority since it's just for local development, but we should be able to run transcripts in parallel to speed things up a bit.

* Would be nice to capture and batch output to print results sensibly (should be able to write from stdout into a buffer rather than direct to terminal)
* I tried just adding `EasyTest.fork` to the appropriate spot in the transcript test builder and it froze on a blocked STM transaction, which probably shouldn't happen, so there may be a bug in easytest
